kubetest/e2e.go

```diff
@@ -201,7 +201,7 @@ func run(deploy deployer, o options) error {
 		}
 	}
 
-	if o.kubemark {
+	if o.up && o.kubemark {
```

Comment:
This would take away the ability of setting up kubemark on an existing cluster.
I don't think there is a gain in in changing this line, I'd revert it.

Reply:
Agree